azure/scope/identity.go

```diff
@@ -182,9 +181,9 @@ func (p *AzureCredentialsProvider) GetAuthorizer(ctx context.Context, resourceMa
 			Name:      fmt.Sprintf("%s-binding", copiedIdentity.Name),
 			Namespace: copiedIdentity.Namespace,
 			Labels: map[string]string{
-				clusterv1.ClusterLabelName:         clusterMeta.Name,
-				infrav1.ClusterLabelNamespace:      clusterMeta.Namespace,
-				clusterctl.ClusterctlMoveLabelName: "true",
+				clusterv1.ClusterLabelName:                   clusterMeta.Name,
+				infrav1.ClusterLabelNamespace:                clusterMeta.Namespace,
+				"clusterctl.cluster.x-k8s.io/move-hierarchy": "true",
```

Comment:
why more away from using a constant here?

Reply:
I'll fix that